.github/workflows/ISSUE_TEMPLATE/pull_request_template.md

```diff
@@ -11,22 +11,10 @@ This PR fixes the following issues:
 
 Here comes all the changes proposed through this PR
 
-<!-- Check all the boxes which are applicable to check the box correct follow the following conventions-->
-<!--
-[x] - Correct
-[X] - Correct
--->
+- change one
+- change two
+- ...
 
-## Check List (Check all the boxes which are applicable) <!--Follow the above conventions to check the box-->
-
-- [ ] My code follows the code style of this project.
-- [ ] My change requires a change to the documentation.
-- [ ] I have updated the documentation accordingly.
-- [ ] All new and existing tests passed.
-- [ ] This PR does not contain plagiarized content.
+## Check List (Check all the boxes that are applicable)
```

Comment:
i didn't mean that you need to remove the whole list here. just the lines i mentioned :)

Reply:
Got it, I'll remove the lines you mentioned and leave the rest of the list as is. Thanks for the heads up!